governance-release-process/process.md

```diff
@@ -35,4 +35,17 @@ In addition to the scheduled releases, hotfix releases of the schemas, i.e. rele
 
 All the releases, including a full changelog listing the changes made, are published (and will remain available) in the [Releases](https://github.com/INSPIRE-MIF/application-schemas/releases) page of this repository. After each release, the INSPIRE schemas at https://inspire.ec.europa.eu/schemas are updated accordingly.
 
+## Schema versioning
+
 Each release will be identified by a sequential number, but the versioning of the INSPIRE schemas will be managed separately.
+
+The versioning will depend on the specific type of schema update (major, minor, bugfix) and will follow rules specified below:
```

Comment:
I would say 'The versioning of the INSPIRE schemas', to be fully sure that readers understand we are speaking about the schema version and not the release version.